File: manualPackage/mappls-map-react-native/ios/RCTMGL/LayerRequest.m
//
//  LayerRequest.m
//  mappls-map-react-native
//
//  Created by Shashank on 3/3/22.
//

#import "LayerRequest.h"

@implementation LayerRequest

-(void)setQuery:(NSString *)query {
    _query = query;
}

-(void)setAttribute:(NSString *)attribute {
    _attribute = attribute;
}

-(void)setGeoBound:(NSArray *)geoBound {
    _geoBound = geoBound;
}

-(void)setPropertyNames:(NSArray *)propertyNames {
    _propertyNames = propertyNames;
}

-(void)setStyleRequest:(StyleRequest *)styleRequest {
    _styleRequest = styleRequest;
}


@end

File: manualPackage/mappls-map-react-native/android/rctmgl/src/main/java/com/mappls/sdk/maps/rctmgl/components/geoAnalytics/LayerRequest.java
package com.mappls.sdk.maps.rctmgl.components.geoAnalytics;

public class LayerRequest {

    private String [] geoBound;
    private String[] propertyNames;
    private String attribute;
    private String query;
    private StyleRequest styleRequest;

    public String[] getGeoBound() {
        return geoBound;
    }

    public void setGeoBound(String[] geoBound) {
        this.geoBound = geoBound;
    }

    public String[] getPropertyNames() {
        return propertyNames;
    }

    public void setPropertyNames(String[] propertyNames) {
        this.propertyNames = propertyNames;
    }

    public StyleRequest getStyleRequest() {
        return styleRequest;
    }

    public void setStyleRequest(StyleRequest styleRequest) {
        this.styleRequest = styleRequest;
    }

    public String getAttribute() {
        return attribute;
    }

    public void setAttribute(String attribute) {
        this.attribute = attribute;
    }

    public String getQuery() {
        return query;
    }

    public void setQuery(String query) {
        this.query = query;
    }
}
